### ISSUE TITLE
[BUG] Date not removable

### ISSUE THREAD

**Issue by Shaah-i** [AmruthPillai/Reactive-Resume#1281]
**Describe the bug**
If I put a date in the date field any where in the resume and then want to remove the date completely it does not get erased.

**Product Flavor**

- [ ] Managed (https://rxresu.me)
- [ ] Self Hosted

**To Reproduce**

1. Go to 'Date of Birth'
2. Fill in 'date of choice'
3. Check the preview if the date has appeared on resume
4. Again go to 'Date of Birth'
5. Delete the filled date
6. Now check the resume preview the date is note deleted.
7. This same is observed for dates in Project, certificatations and all other sections.

**Desktop (please complete the following information):**

- OS: Windows
- Browser chrome
- Version --

**Comment by ja2ui0** [comment]
This was the first thing I noticed trying this out today. I think a simple visibility eyeball like for some of the other sections would be great!

**Comment by vaz** [comment]
Yeah beyond just a :+1: I will add that this was my experience with my first attempt at creating a resume today, and the only solution (besides hacking CSS maybe) was to trash it and start a new one. It seems like a minor thing on one hand but it could probably cause some people to just give up on the app in that scenario, especially if they'd put more time into that resume.

**Comment by AmruthPillai** [comment]
This issue was automatically closed as it was created before 2024-01-01.
